roles/container-engine/containerd/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Download and unarchive containerd
```

Comment:
All download tasks should be in the downloads role

Reply:
Basically, I agree with you. 
Unfortunately, some of download contains require container engine. Especially, container image used by kubeadm. 
Then currently, containerd and related binary must be installed before run download tasks.

Should we keep this? Or create 'pre-download  role' to download container engine split to container image dependency?

Reply:
now PR use containerd.io package is's included in docker-ce and I removed containerd's donwload related code.

For crictl, I  prefer PR #4629 's code. Please merge #4629 first, then I'll change this PR.

Reply:
#4629 is already included this PR.

Reply:
Don't need containerd download code. Now docker-ce's or PPA's containerd.